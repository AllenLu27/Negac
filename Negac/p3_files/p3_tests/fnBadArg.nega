void func(bool b)
{
  return;
}

int main()
{
  int a;
  func(a);
  return 0;
}
