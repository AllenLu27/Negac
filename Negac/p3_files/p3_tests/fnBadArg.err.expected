FATAL [9,8]: Type of actual does not match type of formal
Semantic analysis failed
